Sprint-3/reading-list/script.js

```diff
@@ -1,3 +1,37 @@
+//create a function to display the books in the reading list
+function displayBooks() {
+  const bookListElement = document.getElementById("reading-list");
+  bookListElement.innerHTML = ""; // Clear the existing list
+
+  books.forEach((book) => {
+    // Create a single <li> element
+    const bookElement = document.createElement("li");
+    
+    // Set background color based on alreadyRead status
+    bookElement.style.backgroundColor = book.alreadyRead ? "lightgreen" : "rgb(245, 43, 43)";
+    
+    // Add text content (title & author), adding in paragraph makes the text more readable and styling easier
+    bookElement.innerHTML = `<p>${book.title}</br> by ${book.author.toUpperCase()}</p>`;
+
+    // Create and configure the image
+    const bookElementCover = document.createElement("img");
+    bookElementCover.src = book.bookCoverImage;
+    bookElementCover.alt = book.title;
+    bookElementCover.style.width = "100px"; // Set image size
+    bookElementCover.style.marginLeft = "10px"; // Add spacing
+
+    // Append the image inside the <li>
+    bookElement.appendChild(bookElementCover);
+    
+    // Append the <li> to the <ul> 
+    bookListElement.appendChild(bookElement);
+  });
+}
+
+
+
```

Comment:
While defining  constants like  bookElementCover, please define them globally, so that your script will be more organized.